### PR TITLE
Update invoices.info and add registerPayment

### PR DESCRIPTION
**Pull request by luisaterrabrito**
Add payments and total_due to invoices.info

### CODE REVIEW

**Review by WouterSioen — APPROVED** [review]

**Review by mrtus — APPROVED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -1142,7 +1142,13 @@ Get details for a single invoice.
                         + rate: 0.21 (number)
                         + taxable (Money)
                         + tax (Money)
+                + total_due: 0 (number)
```
Comment (jenssegers) [inline apiary.apib]: Not a big fan of this naming. Would rather have this just `due`.

File: apiary.apib [hunk 2]
```diff
@@ -1256,6 +1262,21 @@ Delete an existing invoice. Only possible for draft invoices or the last booked
...
 + Response 204
 
+### invoices.registerPayment [POST /invoices.registerPayment]
+
+Register a payment for an invoice.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `7abb325c-e063-42a4-8fb4-1b730759645a` (string, required)
```
Comment (jenssegers) [inline apiary.apib:1272]: I thought we discussed we would register payments using a reference rather than introducing payment uuids?

**Review by jenssegers — DISMISSED** [review]
Can you check my remarks?

File: apiary.apib (hunk 1 above)
Comment (luisaterrabrito) [reply apiary.apib]: will change this

 :ok: by @luisabritotl on 31/01/2018 14:24:38

File: apiary.apib (hunk 2 above)
Comment (luisaterrabrito) [reply apiary.apib:1272]: I thought we were going to use the uuid, will bring up the discussion in the channel


 :ok: by @luisabritotl on 31/01/2018 14:44:57

File: apiary.apib (hunk 1 above)
Comment (WouterSioen) [reply apiary.apib]: I guess this change will also need to be reflected in the end-to-end tests?

Comment (luisaterrabrito) [reply apiary.apib]: done

 :ok: by @luisabritotl on 31/01/2018 14:44:49

File: apiary.apib (hunk 2 above)
Comment (stivni) [reply apiary.apib:1272]: btw, this `id` is the invoice id :p

 :ok: by @stivni on 01/02/2018 17:03:54

File: apiary.apib [hunk 3]
```diff
@@ -1142,7 +1142,13 @@ Get details for a single invoice.
                         + rate: 0.21 (number)
                         + taxable (Money)
                         + tax (Money)
+                + due: 0 (number)
```
Comment (stivni) [inline apiary.apib:1145]: Q : doesn't `due` need to be added to the `.list` call too?

Comment (luisaterrabrito) [reply apiary.apib:1145]: I guess, but it isn't there yet

**Review by jenssegers — APPROVED** [review]

**Pull request merged**